apps/oxlint/src/lint.rs

```diff
@@ -375,13 +377,7 @@ impl CliRunner {
             None
         };
 
-        match lint_runner.lint_files(
-            &files_to_lint,
-            tx_error.clone(),
-            file_system
-                .as_ref()
-                .map(|b| &**b as &(dyn oxc_linter::RuntimeFileSystem + Sync + Send)),
```

Comment:
this looked suspicious in the original PR, but i assumed it was fine since CI was green - whoops